File: is28/shishmakova28/Python/lab2/prog2.py
/*import math
print('Введите координаты первой точки')
x1=int(input())
y1=int(input())
print('Введите координаты второй точки')
x2=int(input())
y2=int(input())
S=(y1-y2)**2+(x1-x2)**2
S=math.sqrt(S)
print('%.3f'%S)*/


using namespace std;
int main()
{
int vozrast, voz10;
cout << "сколько вам лет?\n";
cin >> vozrast;
voz10 = vozrast+10;
cout << "Через 10 лет вам бóдет: " << voz10;
return 0;
}

File: is28/vyacheslavleva28/lab12/prog2.cpp
#include <math.h>
#include <iostream>

using namespace std;

int main()
{
	int x;
	int a1; 
	int a2;
	int a3;
	
	cin >> x;
	
	a1 = x % 10;
	a2 = x / 10;
	a2 = a2 % 10;
	a3 = x / 100;
	
	cout << (a1 + a2 + a3) << endl;
	
	return 0;
	
}
